blackbox/docs/administration/auth_methods.txt

```diff
@@ -0,0 +1,18 @@
+
+.. _auth_methods:
+
+======================
+Authentication Methods
+======================
+
+.. _auth_trust:
+
+Trust Method
+============
+
+When the ``trust`` method is used, CrateDB assumes that anyone is allowed access
+to the cluster. Therefore, this method allows any client to login as any existing
+CrateDB user (even the ``crate`` superuser) without specifying further credential
+information like passwords etc.
+
+The ``trust`` method can be used in conjunction with the :ref:`administration_hba`.
```

Comment:
@dobe I don't understand this comment : ) do you mean I should add a paragraph about when it is best to use this method ?